src/EnergyPlus/TARCOGParams.hh

```diff
@@ -65,39 +65,56 @@ int constexpr maxlay2(maxlay * 2);  // maximum number of glass surfaces (maxlay*
 int constexpr maxlay3(maxlay2 + 1); // maximum number of ? (maxlay2+1)
 
 //  Layer types:
-extern int const SPECULAR;
-extern int const VENETBLIND_VERT;
-extern int const WOVSHADE;
-extern int const PERFORATED;
-extern int const DIFFSHADE;
-extern int const BSDF;
-extern int const VENETBLIND_HORIZ;
-
-extern int const MinLayType;
-extern int const MaxLayType;
+enum class TARCOGLayerType : int
+{
+    UNASSSIGNED = -1,
+    SPECULAR = 0,
+    VENETBLIND_HORIZ = 1,
+    WOVSHADE = 2,
+    PERFORATED = 3,
+    DIFFSHADE = 4,
+    BSDF = 5,
+    VENETBLIND_VERT = 6
+};
 
```

Comment:
So ... in some places we are setting `UNASSIGNED` as `0`, in others we are using `-1`. We should settle on one, preferably `-1` but we could make `0` work also.

Reply:
Got it. Will change to `-1`, and let the group know as well 👍🏽